src/components/pluginManager.js

```diff
@@ -74,14 +74,23 @@ import { playbackManager } from './playback/playbackmanager';
             if (typeof pluginSpec === 'string') {
                 if (pluginSpec in window) {
                     console.log(`Loading plugin (via window): ${pluginSpec}`);
-                    let pluginInstance = await window[pluginSpec];
 
-                    if (typeof pluginInstance === 'function') {
-                        pluginInstance = await new pluginInstance();
+                    const pluginDefinition = await window[pluginSpec];
+                    if (typeof pluginDefinition !== 'function') {
+                        const err = new TypeError('Plugin definitions in window have to be an (async) function returning the plugin class');
+                        console.error(err);
```

Comment:
Do we need to log here when we log errors when an error is thrown by this function?

Reply:
I did it that way because further down the same function the existing code did the same. Don't know if it's necessary.

Reply:
It's in fact not necessary, pushed a commit removing the extra logging calls.